builder/ecs/run_config.go

```diff
@@ -150,7 +170,13 @@ type RunConfig struct {
 
 type DataVolume struct {
 	// The data disk size in GB.
-	Size int `mapstructure:"volume_size" required:"true"`
+	Size int `mapstructure:"volume_size" required:"false"`
+	// The ID of the image.
+	DataImageId string `mapstructure:"data_image_id" required:"false"`
+	// The ID of the snapshot.
+	SnapshotId string `mapstructure:"snapshot_id" required:"false"`
+	// The ID of the volume.
```

Comment:
how about using `The ID of an existing Volumes` ?

Reply:
done